### ISSUE TITLE
Unable to open .nuspec file via Visual Studio's 'Open With...' after installing W10 Store version

### ISSUE THREAD

**Issue by rjgotten** [NuGetPackageExplorer/NuGetPackageExplorer#220]
**Type** : Bug
**NPE version**: 3.23.0.0
**Installed from**:  Windows Store

**Current result**
After installing via the Windows Store, I am attempting to open a `.nuspec` file directly from Visual Studio's Solution Explorer, via the `Open With...` command in the right-click context menu.
The dialog lists `NuGet Package Explorer` as a viable option, however attempting to open the file into NPE that way shows the error "The file cannot be opened with the selected editor. Please choose another editor."

**Expected result**
The `.nuspec` file should open in NPE when using Visual Studio's `Open With...` command.

**Workarounds**
Using an `Open Containing Folder...` command to launch Explorer and then double-clicking the `.nuspec` file from there to launch it with the default file handler adds a few steps, but works.

**Comment by hardcodet** [comment]
Also, adding a new feed in VS2017 crashes VS with the explorer installed. Uninstalling the app from Windows Store fixed the issue.

**Comment by 304NotModified** [comment]
> Also, adding a new feed in VS2017 crashes VS with the explorer installed. Uninstalling the app from Windows Store fixed the issue.

please post this a separate issue and add the steps to reproduce this. I never experienced that problem

**Comment by arteny** [comment]
Issue still reproduced for NPE 3.24.0.0

**Comment by onovotny** [comment]
I can repro with the latest release. Looking into this.

**Comment by riverar** [comment]
This is a shell/visual studio bug. Visual Studio relies on `AssocQueryString` to resolve Open With progids. But there's no "executable" associated with NuGet Package Explorer (or Appx apps in general), so it returns an error and Visual Studio gets confused.

**Comment by onovotny** [comment]
@riverar sounds like a Visual Studio issue then?

**Comment by riverar** [comment]
@onovotny Yeah, it's a tough one. UWP apps are a bit shoehorned into OpenWithProgIds; consumers weren't exactly expecting that to need special consideration for UWP apps. I'd blame appmodel 🤣

Have you tried adding an app execution alias? That might change the way these are registered and may Just Work ™️

**Comment by onovotny** [comment]
@riverar I tried that without luck. It's in #324, which is a good idea to have anyway.
